docs/howto_subplot.rst

```diff
@@ -72,3 +72,45 @@ subplots, as in some cases there is no need to repeat the tick labels. Alternati
     # some code here with ax2
 
     fig.canvas.draw()
+
+Layouts can also be specified by the use of tuples of the form
+(numrows, numcols, fignum) in the same manner as MATLAB and matplotlib.
+
+Complex layouts can be specified with the use of ``GridSpec`` objects. For
+example, the following code makes a 9 by 9 grid of axes, and uses a 2 by 2
+slice (``gs[1:3, 0:2]``) to display a figure.::
+
+    from astropy.io import fits
```

Comment:
You can use the ``.. plot`` directive here to automatically include the result in the docs